atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1773,6 +1774,27 @@ private void alterGcAndTombstoneWithLock(
         }
     }
 
+    public void cleanUpSchemaMutationLockTablesState() throws Exception {
+        // get a unique table and create it if it doesn't exist
+        Set<TableReference> tables = lockTables.getAllLockTables();
+        TableReference unique;
+        if (tables.size() == 0) {
+            unique = lockTables.createLockTable();
+            log.info("There were no locks tables, so one was succesfully created: {}", unique.toString());
+            return;
+        }
+        unique = tables.iterator().next();
+        tables.remove(unique);
```

Comment:
This should work because, as far as I can tell, `dropTablesWithLock` verifies that the schema change took effect on _all_ nodes (via `waitForSchemaVersions`).  If there is a risk `dropTablesWithLock` would return even when some nodes were unable to process the drop, then disregard what I said above.